docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,415 @@
+
+When you download the Prebid.js package, add the UID2 module by checking the box next to the module named **Unified ID 2.0**, listed under the section **User ID Modules**.
+
+When you've added Prebid.js to your site and confirmed that it's working properly, you're ready to configure the UID2 module.
+
+:::tip
+To make sure that the UID2 module is installed, find the string `uid2IdSystem` in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::
+
+## Configure the UID2 Module
+
+You'll need to configure the UID2 Prebid module to complete the following two actions:
+
+| Step | Action | Link to Instructions |
+| --- | --- | --- |
+| 1 | Send a server-side API call to generate a UID2 token.  | [Generating a UID2 Token on the Server](#generating-a-uid2-token-on-the-server) |
+| 2 | Store the response value, so that the Prebid module can manage token refresh as well as opt-out if needed. | [Refreshing a UID2 Token](#refreshing-a-uid2-token) |
+
+### Generating a UID2 Token on the Server
+
+To generate a token, call the [POST /token/generate](../endpoints/post-token-generate.md) endpoint.
+
+For an example, see [Sample Token](#sample-token).
+
+### Refreshing a UID2 Token
+
+There are two ways to refresh a UID2 token, as shown in the following table.
+
+| Mode | Description | Link to Section | 
+| --- | --- | --- |
+| Client refresh mode | Prebid.js automatically refreshes the tokens internally.<br/>This is the simplest approach. | [Client Refresh Mode](#client-refresh-mode) |
+| Server-only mode | Prebid.js doesn't automatically refresh the token. It is up to the publisher to manage token refresh.<br/>Examples of why you might want to choose this option:<ul><li>If you want to use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md) to refresh the token, and Prebid.js to send the token to the bid stream.</li><li>If you want to send the token to the bid stream via multiple avenues (such as Prebid.js and also Google).</li></ul> | [Server-Only Mode](#server-only-mode) |
+
+### Client Refresh Mode
+
+You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+
+- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
+- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+
+For an example, see [Sample Token](#sample-token).
+
+As long as the refresh token remains valid, the UID2 Prebid module refreshes the UID2 token as needed.
+
+This section includes the following information:
+- [Client Refresh Mode Response Storage Options](#client-refresh-mode-response-storage-options)
+- [Client Refresh Mode Cookie Example](#client-refresh-mode-cookie-example)
+- [Client Refresh Mode uid2Token Example](#client-refresh-mode-uid2token-example)
+- [Passing a New Token: Client Refresh Mode](#passing-a-new-token-client-refresh-mode)
+
+#### Client Refresh Mode Response Storage Options
```

Comment:
I think the term "storage" here is misleading. It's about how the publisher provides the data to the module, and it's unrelated to how the module stores the data. Perhaps a better title would be:

> Client Refresh Mode Response Configuration Options

Reply:
Done, thx.

Reply:
Nice.